src/style.css

```diff
@@ -1,3 +1,109 @@
-body {
+/* General reset */
+* {
   margin: 0;
 }
+
+body {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+
+  --color-blue: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo__link {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.menu {
+  display: flex;
+  padding-left: 0;
+}
+
+.menu__item {
+  list-style: none;
+  align-items: center;
+  display: flex;
+}
+
+.menu__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.menu__link {
+  text-decoration: none;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  color: #000;
+  height: 60px;
+  position: relative;
+  font-weight: 500;
+}
+
+ .is-active,
+.menu__link:hover {
+  color: var(--color-blue);
+}
+
+.menu__link.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0; /* Ustawiamy na wartość ujemną, aby podkreślenie było "na zewnątrz" tekstu */
+  left: 0;
+  right: 0;
+  width: 100%;
+  height: 4px; /* Grubość podkreślenia */
+  background-color: var(--color-blue);
+  border-radius: 2px; /* Zaokrąglamy podkreślenie */
+}
+
+/* @media screen and (max-width: 1024px) {
+  .menu {
+    position: relative;
+  }
+
+  .menu::before {
+    content: "";
+    display: none;
+    position: absolute;
+    top: 100%;
+    left: 0;
+    width: 100%;
+    height: 1px;
+    background-color: #ddd;
+    z-index: -1;
+  }
+
+  .menu-item {
+    padding: 10px 15px;
+  }
+
+  .menu-item:not(:last-child) {
+    border-right: none;
+    border-bottom: 1px solid #ddd;
+  }
+
+  .menu-item:hover {
+    display: block;
+    position: absolute;
+    top: 100%;
+    left: 0;
+    width: 100%;
+    padding: 10px 15px;
+    background-color: #fff;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  }
+} */
```

Comment:
remove commented lines,
it's a good practice to remove all unnecessary comments and console.logs before commit